hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -1037,24 +1049,34 @@ synchronized void internalGetSSTDiffList(
             continue;
           }
 
-          for (CompactionNode node : successors) {
-            if (sameFiles.contains(node.getFileName()) ||
-                differentFiles.contains(node.getFileName())) {
-              LOG.debug("Skipping known processed SST: {}", node.getFileName());
+          for (CompactionNode nextNodes : successors) {
```

Comment:
nit: variable node could be better than nextNodes & successors. Probably sourceNode & sourceNodes

Reply:
Since this is a dag successor doesn't really sound good. But that is fine though